nilearn/input_data/nifti_labels_masker.py

```diff
@@ -107,6 +108,11 @@ class NiftiLabelsMasker(BaseMasker, CacheMixin):
     verbose: integer, optional
         Indicate the level of verbosity. By default, nothing is printed
 
+    func: str
```

Comment:
I propose calling this `strategy`, `reduce_using`, `aggregate_using`,`summary_function` ... something along such lines. The variable name should immediately indicate what the basic purpose/value is without the need for full context.

Reply:
I like `strategy`. Even though it's not immediate what it means, it's easier to remember and to write correctly on the first attempt.